# 07 Crack spread

Does a richer 3-2-1 crack line up with a higher five-print RWTC return, and does that sign survive 2020?

Holdout starts 2020-01-01. Verdict: failed.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'crack')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
Train correlation of the 3-2-1 crack with the next five-print RWTC return is 0.036. Holdout correlation is -0.030. The holdout flipped the train sign.
CRACK_321 level versus the next five RWTC prints
Train | 3371 | 0.036
Holdout | 1066 | -0.030


## Method

The theory under test is simple: a wide crack pulls refinery demand for crude, so the next RWTC return should be positive when the crack is high. The score is the correlation of the crack level with the five-print RWTC log return.

## Result

Train correlation of the 3-2-1 crack with the next five-print RWTC return is 0.036. Holdout correlation is -0.030. The holdout flipped the train sign.